server/tso.go

```diff
@@ -39,21 +39,23 @@ func (s *Server) getTimestampPath() string {
 	return path.Join(s.rootPath, "timestamp")
 }
 
-func (s *Server) loadTimestamp() (int64, error) {
+func (s *Server) loadTimestamp() (time.Time, error) {
+	errTime := time.Unix(0, 0)
```

Comment:
Why not using time.Time{} for invalid return value, then we can use IsZero for check?